_data/companies.yml

```diff
@@ -110,13 +110,6 @@
   events: Restricted 
   last_update: 2020-03-12  
 
-- name: Bitwala 
-  wfh: Encouraged 
-  travel: Restricted 
-  visitors: "?" 
-  events: Restricted 
-  last_update: 2020-03-12 
-
```

Comment:
It looks like your git client tried to delete this entry. Would you mind adding it back?

Thanks!

Reply:
Done